src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbTestFixture.scala

```diff
@@ -37,6 +37,63 @@ trait OrientDbTestFixture extends BeforeAndAfterAll {
     def save(txn:RawlsTransaction)
   }
 
+  class EmptyWorkspace() extends TestData {
```

Comment:
It's fine to keep it, I was just wondering if it was supposed be used but wasn't.